client/src/pages/Login.js

```diff
@@ -175,6 +206,7 @@ const Login = ({ isLoginForm }) => {
   );
   const [passwordType, setPasswordType] = useState("password");
   const [confirmPasswordType, setConfirmPasswordType] = useState("password");
+  const [sentRecoveryLink, setRecoveryLink] = useState(false);
```

Comment:
```suggestion
  const [recoveryLink, setRecoveryLink] = useState(false);
```

Seems to be the react naming convention

Reply:
Pushed changes for the fix commit 6e7fefe.